.github/workflows/ci.yaml

```diff
@@ -21,12 +21,17 @@ jobs:
         env:
           VERTICA_CE_URL: "https://vertica-community-edition-for-testing.s3.amazonaws.com/XCz9cp7m/vertica-12.0.0-0.x86_64.RHEL6.rpm"
         run: |
-          git clone https://github.com/jbfavre/docker-vertica.git
-          curl $VERTICA_CE_URL --create-dirs -o docker-vertica/packages/vertica-ce.latest.rpm
-          docker build -f docker-vertica/Dockerfile.centos.7_9.x --build-arg VERTICA_PACKAGE=vertica-ce.latest.rpm -t jbfavre/vertica docker-vertica
-          docker images
-          docker run -d -p 5433:5433 jbfavre/vertica
-          sleep 60
+          docker pull vertica/vertica-ce:12.0.0-0
```

Comment:
This looks much more maintainable.   I think it might be worth putting a comment in explaining that -d runs the container in the background.   I'm also curious about the two ports.   Why -p for 5433 and 5444?